react/daily-art-prompt/src/components/PromptLayout.test.js

```diff
@@ -36,6 +41,35 @@ describe('<PromptLayout>', () => {
       })
     })
 
+    describe('<Prompt>', () => {
+      describe('when prompts.length is greater than 0', () => {
+        it('should render', () => {
+          const wrapper = shallow(<PromptLayout {...defaultProps} />);
+          const result = wrapper.find({ 'data-testid': 'prompt' })
+          console.log('result is: ', result)
+          expect(result).toHaveLength(1)
+        })
+        it('should render with correct props', () => {
```

Comment:
do you mean it has the correct prompt prop?

Reply:
yes i did